tests/repositories/test_http_repository.py

```diff
@@ -153,3 +153,33 @@ def test_get_info_from_wheel_state_sequence(mocker: MockerFixture) -> None:
     repo._get_info_from_wheel(link)
     assert mock_metadata_from_wheel_url.call_count == 5
     assert mock_download.call_count == 4
+
+
+@pytest.mark.parametrize(
+    "mock_hashes",
+    [
+        None,
+        {"sha256": "e216b70f013c47b82a72540d34347632c5bfe59fd54f5fe5d51f6a68b19aaf84"},
```

Comment:
Shouldn't this be a case where repo does not provide a sha256?

Reply:
Since there were no unit tests for this function as far as I could see, the unit test will run with 3 cases (no hashes in link, sha256 in link, and md5 in link)